BrainPortal/app/views/access_profiles/show.html.erb

```diff
@@ -53,7 +53,7 @@
       <%= render :partial => 'shared/color_picker', :locals => { :step => 30, :greys => false, :dark => false } %>
     <% end %>
 
-    <% t.edit_cell(:description, :content => overlay_description(@access_profile.description), :show_width => 2 ) do %>
+    <% t.edit_cell(:description, :content => @access_profile.description ) do %>
```

Comment:
The way the description is shown no longer matches the formatting I entered int he text area box.